File: src/main/java/fr/vlegall/sochief/SoChiefApplication.java
package fr.vlegall.sochief;

import fr.vlegall.sochief.configuration.SecurityApiProperties;
import fr.vlegall.sochief.configuration.minio.MinioProperties;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.EnableConfigurationProperties;

@EnableConfigurationProperties({
        MinioProperties.class,
        SecurityApiProperties.class
})
@SpringBootApplication
public class SoChiefApplication {
    public static void main(String[] args) {
        SpringApplication.run(SoChiefApplication.class, args);
    }
}

File: src/main/kotlin/fr/vlegall/sochief/SoChiefApplication.kt
package fr.vlegall.sochief

import fr.vlegall.sochief.configuration.minio.MinioProperties
import fr.vlegall.sochief.configuration.SecurityApiProperties
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.EnableConfigurationProperties
import org.springframework.boot.runApplication

@EnableConfigurationProperties(
    value = [
        MinioProperties::class,
        SecurityApiProperties::class
    ]
)
@SpringBootApplication
class SoChiefApplication

fun main(args: Array<String>) {
    runApplication<SoChiefApplication>(*args)
}
